### ISSUE TITLE
Publish methane manure collection(s) to production

### ISSUE THREAD

**Issue by anayeaye** [NASA-IMPACT/veda-data#172]
## What
This issue is to collect the remaining data tasks required to publish the methane emissions dataset(s) to the earthdata dashboard.

**Blocked PR** [veda-config/Initial Hog Farm Push #380](https://github.com/NASA-IMPACT/veda-config/pull/380)

## Tasks
- [ ] Finish validating and updating collections in staging backend
- [ ] Transfer files to production
- [ ] Publish collections to production
- [ ] Open veda-data PR with validated publication configuration

## Raw config info (edit and publish in staging)
An older veda-data PR ([fix/draft1-methane-manure-collection](https://github.com/NASA-IMPACT/veda-data/pull/120)) has the starting work on the dataset configuration for publishing these collections to staging and production--it can be used as a starting place but this PR is so out of date that we should start a new PR for the work. 

## Unblocked veda-config PR
Now [veda-config/Initial Hog Farm Push #380](https://github.com/NASA-IMPACT/veda-config/pull/380) can be updated as needed and published to production

**Comment by anayeaye** [comment]
The emission collections are already in the GHG catalog and available to the dashboard from there so these VEDA ingest is not necessary.
